Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py

```diff
@@ -200,6 +197,7 @@ def main() -> None:
     reliability = params.get('integrationReliability')
 
     demisto.debug(f'Command being called is {command}')
+    demisto.debug(f'Is time sensitive: {is_time_sensitive()}')
```

Comment:
let's call `is_time_sensitive` only once (server call), then pass its result to the Client as an attribute